### PR TITLE
[PyTorch] Add sigmoid op

### PR DESCRIPTION
**Pull request by zrphercule**
Summary:

Documentation:

[Optional Fixes #issue]

Test Plan:

Please see a detailed explanation of how to fill out the fields in the relevant sections in PULL_REQUEST.md.

### CODE REVIEW

File: torch_glow/tests/nodes/sigmoid_test.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    jitVsGlow(sigmoid_basic, x)
+
+def test_sigmoid_inplace():
+    def sigmoid_inplace(a):
+        c = a.sigmoid_()
+        return c.sigmoid_()
+
+    x = torch.randn(6)
+
+    jitVsGlow(sigmoid_inplace, x)
```
Comment (yinghai) [inline torch_glow/tests/nodes/sigmoid_test.py]: We support inplace because we copy Glow's output back to `x`?

Comment (zrphercule) [reply torch_glow/tests/nodes/sigmoid_test.py]: In this case, I think the answer is yes, although I didnt see any using scenario

File: torch_glow/tests/nodes/sigmoid_test.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import torch
+import torch_glow
+
+from tests.utils import jitVsGlow
+
+# Basic test of the PyTorch sigmoid Node on Glow
+
+
+def test_sigmoid_basic():
+    def sigmoid_basic(a):
```
Comment (jackm321) [inline torch_glow/tests/nodes/sigmoid_test.py]: I started just calling these `test_f` so I don't need to come up with a new name each time, you can do the same or not up to you

File: torch_glow/src/PyTorchModelLoader.cpp [hunk 3]
```diff
@@ -226,6 +226,7 @@ struct LinearInputs {
 // static
 const PyTorchModelLoader::MappingOfMemberFunctions &
 PyTorchModelLoader::getSymbolsMapping() {
+
```
Comment (jackm321) [inline torch_glow/src/PyTorchModelLoader.cpp]: ?
